README.md

```diff
@@ -55,28 +62,34 @@ the [Swagger documentation](https://lps-oppfolgingsplan-mottak.ekstern.dev.nav.n
+
+Please refer
+to [NAV's guide for testing API's](https://github.com/navikt/nav-ekstern-api-dok/blob/main/api-dok/teste-api/teste-api.md)
 
 ## Error Codes
-If you get an error, we will provide an ApiError object in the response body. 
+
+If you get an error, we will provide an ApiError object in the response body.
 An ApiError will contain the following fields:
 <pre>
 - status: HttpStatusCode
 - type: ErrorType
 - message: String
 </pre>
 
-| Status | Type                           | Example Message | Description                                                                                               |
-|--------|--------------------------------|---------------|-----------------------------------------------------------------------------------------------------------|
-| 400    | VALIDATION_ERROR               | Invalid employee identification number | The follow-up plan DTO has validation errors. Please check message for the specific error.                |
-| 401    | AUTHENTICATION_ERROR           | JWT is expired | The maskinporten-token is invalid. Please check message for the specific error.                           |
-| 404    | GENERAL_PRACTITIONER_NOT_FOUND | General practitioner was not found | The general practitioner was not found. Please ensure that it is correctly registered for the employee.   |
-| 404    | EMPLOYEE_NOT_FOUND             | Could not find requested person in our systems | This employeeIdentificationNumber is not registered in NAV's systems.                                |
-| 404    | FOLLOWUP_PLAN_NOT_FOUND        | The follow-up plan with a given uuid was not found | The follow-up plan with a given uuid was not found. Only relevant for the status-endpoint.                |
-| 500    | INTERNAL_SERVER_ERROR          | Internal server error | An unexpected error occurred on the server. Please contact the NAV developer team if the problem persists. |
+| Status | Type                           | Example Message                                    | Description                                                                                                |
+|--------|--------------------------------|----------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| 400    | VALIDATION_ERROR               | Invalid employee identification number             | The follow-up plan DTO has validation errors. Please check message for the specific error.                 |
+| 401    | AUTHENTICATION_ERROR           | JWT is expired                                     | The maskinporten-token is invalid. Please check message for the specific error.                            |
+| 403    | NO_ACTIVE_SENT_SYKMELDING      | No active sykmelding sent to employer              | There is no active sykmelding, or the sykmelding is not sent to the employer                               |
```

Comment:
Litt usikker om `403` passer best her.

Reply:
Litt bekymret at 4030gir inntrykk at det er noe galt med selve tilganger, osv. (403 Forbidden)
Kanskje den er også `400`?

Reply:
Ja var litt usikker på hva som blir best. Dersom ikke bedriften har en sykmelding så har man ikke lov til å sende inn på vegne av den ansatte, så derfor tenkte jeg 403.

Tenkte også litt på 404, ala "sykmelding not found", men copilot mente 403 var best :P 

400 går sikkert også.

Hva tenker du @Vitenok ?

Reply:
ChatGPT mente til meg 400 😆 . Luring 🤣

Reply:
Haha, Gemini støtter 403 😅

Reply:
Lurer på om det burde også være 403 for manglende fastlege 🤔

Reply:
tenker at ved manglende plan ønsker vi virkelig å understreke at det er brudd på forretningsregel da vi KAN sende plan, men har ikke lov til å gjøre det.
Mens i tilfelle manglende fastlege det er manglende resource og planen kan IKKE sendes fysiskt. Da er det greit med 404.
Mener fortsatt at 400 blir misvisende, fordi request er riktig.